assembl/static2/js/app/components/debate/thread/post.jsx

```diff
@@ -90,6 +95,31 @@ class Post extends React.PureComponent {
     } = this.props.data.post;
     const { lang, ideaId, refetchIdea, creationDate, fullLevel, numChildren } = this.props;
     // creationDate is retrieved by IdeaWithPosts query, not PostQuery
+
+    let body;
+    let subject;
+    let originalBodyLocale;
+    let originalBody;
+    let originalSubject;
+    if (bodyEntries.length > 1) {
```

Comment:
Maybe but this block in a helper function (it appears at least thrice from what I've seen)

Reply:
Yeah, it appear twice I think, in survey post and in thread post, we can try to refactor this for the final version.

Reply:
I mean twice here (body and subject) and once in survey post

Reply:
👍